root/static/scripts/tests/release-editor/fields.js

```diff
@@ -55,11 +55,11 @@ fieldTest("mediums having their \"loaded\" observable set correctly", function (
 
     mediums([
         new fields.Medium({ tracks: [] }),
-        new fields.Medium({ tracks: [ {} ] }),
+        new fields.Medium({ tracks: [{}] }),
         new fields.Medium({ id: 1, tracks: [] }),
         new fields.Medium({ originalID: 1, tracks: [] }),
-        new fields.Medium({ id: 1, tracks: [ {} ] }),
-        new fields.Medium({ originalID: 1, tracks: [ {} ] })
+        new fields.Medium({ id: 1, tracks: [{}] }),
+        new fields.Medium({ originalID: 1, tracks: [{}] })
```

Comment:
I’m not sure these changes are making the code more readable.

Reply:
I dunno, for me they are, since they make it look the same as usual. If you want to show specifically that these are empty, we could add a comment? `[{/* empty */}]` or something? But I'm not sure there's really any doubt here.

Reply:
There's an "objectsInArrays" exception for https://eslint.org/docs/rules/space-in-brackets but you can only enforce one style or the other, not allow both without disabling space-in-brackets entirely AFAICT.

Reply:
Personal preference mostly, the new form seems more likely confused with `[]`.
But as told on IRC, that would be even more cumbersome to work around eslint here.